client/index.html

```diff
@@ -59,73 +59,66 @@
     <script src="app/environments/dialogs/asg/asgLaunchConfig.js"></script>
     <script src="app/environments/dialogs/asg/asgInstances.js"></script>
     <script src="app/operations/operations.module.js"></script>
+    <script src="app/operations/upstream/directives/lbServersStatesCell.js"></script>
```

Comment:
Lots of changes in this file due to gulp task, but this is the only change. Added lbServersStatesCell.js